ckanext/hierarchy/plugin.py

```diff
@@ -141,6 +140,17 @@ def before_dataset_search(self, search_params):
             # remove include_children from the filter-list - we have a checkbox
             g.fields_grouped.pop('include_children', None)
 
+        if 'groups' in search_params['fq']:
+            group_selected = model.Group.get(search_params['fq'].split('"')[1])
+            group_with_children = ' OR '.join(
+                '{}'.format(grp.name)
```

Comment:
just use group name directly here, you won't get anything out of formatting

Reply:
Fixed